src/decision/dMCDA.jl

```diff
@@ -412,22 +415,31 @@ function create_seed_matrix(
     wt_heat::T,
     wt_predec_seed::T,
     wt_predec_zones_seed::T,
-    wt_lo_cover::T
+    wt_low_cover::T
 )::Tuple{Matrix{Float64}, Vector{Float64}} where {T<:Float64}
     # Define seeding decision matrix, based on copy of A
     SE = copy(A)
 
-    wse = [wt_in_conn_seed, wt_out_conn_seed, wt_waves, wt_heat, wt_predec_seed, wt_predec_zones_seed, wt_lo_cover, wt_heat]
-    # wse .= mcda_normalize(wse)
+    wse = [
+        wt_in_conn_seed,
+        wt_out_conn_seed,
+        wt_waves,
+        wt_heat,
+        wt_predec_seed,
+        wt_predec_zones_seed,
+        wt_low_cover,
+        wt_heat,
+    ]
 
-    SE[:, 4] = (1 .- SE[:, 4]) # compliment of wave risk
-    SE[:, 5] = (1 .- SE[:, 5]) # compliment of heat risk
+    SE[:, 4] = (1 .- SE[:, 4])  # compliment of wave risk
+    SE[:, 5] = (1 .- SE[:, 5])  # compliment of heat risk
 
     # Coral real estate as total area, sites with ≤ min_area to be seeded available filtered out
-    SE[vec(A[:, 8] .<= min_area), 8] .= NaN
+    # This will also filter out sites with 0 space
+    SE[SE[:, 8] .<= min_area, 8] .= NaN
 
     # Mark "hot" locations for filter
-    SE[vec(A[:, 5] .>= 0.75), 5] .= NaN
+    #SE[vec(A[:, 5] .>= 0.75), 5] .= NaN
```

Comment:
In terms of impact, this change is very big and unrelated to the purpose of the PR.

Could you shift this to a different PR please.